spec/roo_on_rails/sidekiq/metrics_worker_spec.rb

```diff
@@ -5,148 +5,164 @@
 
   describe '#perform' do
     let(:perform) { described_class.new.perform }
+
     before do
-      allow(statsd).to receive(:batch).and_call_original
-      allow(statsd).to receive(:gauge)
+      allow(statsd).to receive(:gauge).with(any_args)
+      allow(Sidekiq::Queue).to receive(:all).and_return(sidekiq_queues) if defined?(sidekiq_queues)
     end
 
     after { reset_queues }
 
     it 'should send all metrics in a batch' do
+      expect(statsd).to receive(:batch).with(no_args).once
```

Comment:
i made this change because usually it's weird to have the execution of the code being tested in `before` block